docs/stdlib/numbers.rst

```diff
@@ -142,9 +142,10 @@ from :eql:type:`str` and :eql:type:`json`.
 
     :index: int integer
 
-    A 16-bit signed integer.
+    Represents a 16-bit signed integer.
 
-    An integer value in range from ``-32768`` to ``+32767`` (inclusive).
+    The value of an :eql:type:`int16` ranges from ``-32768`` to ``+32767``
+    (inclusive).
```

Comment:
```suggestion
    :eql:type:`int16` is capable of representing values from ``-32768`` to
    ``+32767``, inclusive.
```

The change could be read to mean that one value of the type covers that entire range. This better represents that the type is _capable_ of representing values within the range.

Recommend propagating this phrasing across all the types' ranges.